crates/next-core/src/next_client/context.rs

```diff
@@ -204,6 +204,9 @@ pub async fn get_client_runtime_entries(
         ProcessEnvAssetVc::new(project_root, filter_for_client(env)).into(),
     )
     .cell()];
+    if let Some(request) = enable_react_refresh {
+        runtime_entries.push(RuntimeEntry::Request(request, project_root.join("_")).cell())
+    };
```

Comment:
What's the reason for this reordering?

Reply:
`bootstrap.ts` file imports another file that contains JSX, and that JSX is being transformed. To evaluate any file with JSX now requires that react-refresh has already set up the global `$RefreshSig$`/`$RefreshReg$`.

Reply:
Done.